### PR TITLE
Add schedule

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
- Add schedule hackathon

### CODE REVIEW

**Comment by christopher-hakkaart** [comment]
@mribeirodantas - can you give this a quick check against the draft program please

File: src/content/events/2023/hackathon-november-2023.md [hunk 1]
```diff
@@ -36,11 +36,72 @@ Be sure to register early to avoid disappointment!
...
+          </tr>
+        </tbody>
+    </table>
+</div>
+
+# Groups
+
+To keep work and discussion during the hackathon manageable, we will organise everyone into groups. You are free to change groups as you wish during the hackathon.
+
+We will coordinate our work and the issues we are working on using a single GitHub project board (we’ll post a link to here closer to the time). There will be a separate tab for each group where will have collated a list of relevant issues.
```
Comment (mribeirodantas) [inline src/content/events/2023/hackathon-november-2023.md:98]:
```suggestion
We will coordinate our work and the issues we are working on using a single GitHub project board (we’ll post a link here closer to the time). There will be a separate tab for each group where will have collated a list of relevant issues.
```

File: src/content/events/2023/hackathon-october-2023.md [hunk 2]
```diff
@@ -38,11 +38,83 @@ The 2022 Barcelona hackathon did sell out. So be sure to register early to avoid
...
+      </tr>
+    </tbody>
+    </table>
+</div>
+
+# Groups
+
+To keep work and discussion during the hackathon manageable, we will organise everyone into groups. You are free to change groups as you wish during the hackathon.
+
+We will coordinate our work and the issues we are working on using a single GitHub project board (we’ll post a link to here closer to the time). There will be a separate tab for each group where will have collated a list of relevant issues.
```
Comment (mribeirodantas) [inline src/content/events/2023/hackathon-october-2023.md:109]:
```suggestion
We will coordinate our work and the issues we are working on using a single GitHub project board (we’ll post a link here closer to the time). There will be a separate tab for each group where will have collated a list of relevant issues.
```

**Review by mribeirodantas — APPROVED** [review]

**Review by mashehu — APPROVED** [review]

**Comment by christopher-hakkaart** [comment]
Thanks @mashehu + @mribeirodantas  🤩

**Pull request merged**